source/features/repo-age.tsx

```diff
@@ -5,29 +5,45 @@ import select from 'select-dom';
 import repoIcon from 'octicon/repo.svg';
 import elementReady from 'element-ready';
 import features from '../libs/features';
-import * as api from '../libs/api';
-import {getRepoGQL, getRepoURL} from '../libs/utils';
+import fetchDom from '../libs/fetch-dom';
+import {getRepoURL} from '../libs/utils';
 
 const dateFormatter = new Intl.DateTimeFormat('en-US', {
 	year: 'numeric',
 	month: 'long',
 	day: 'numeric'
 });
 
-const getRepoCreationDate = cache.function(async (): Promise<string> => {
-	const {repository} = await api.v4(`
-		repository(${getRepoGQL()}) {
-			createdAt
-		}
-	`);
+const getFirstCommitDate = cache.function(async (): Promise<string | undefined> => {
+	const lastCommitHash = (await elementReady<HTMLAnchorElement>('.commit-tease-sha', {stopOnDomReady: false}))!.href.split('/').pop()!;
+	const commitsCount = Number(select('li.commits .num')!.textContent!.replace(',', ''));
 
-	return repository.createdAt;
+	if (commitsCount === 0) {
+		return;
+	}
+
+	if (commitsCount === 1) {
+		return select('.commit-tease-sha + span relative-time')!.getAttribute('datetime')!;
+	}
+
+	const oldestCommit = await fetchDom(
+		`${getRepoURL()}/commits?after=${lastCommitHash}+${commitsCount - 2}`, 'ol:last-child > li.commits-list-item'
+	);
+
+	return select('relative-time', oldestCommit)!.getAttribute('datetime')!;
```

Comment:
Can you check if this element also offers a `.datetime` prop, so maybe this works:

```suggestion
	return oldestCommit.datetime;
```